builtin/providers/cloudstack/resource_cloudstack_network.go

```diff
@@ -167,6 +171,13 @@ func resourceCloudStackNetworkRead(d *schema.ResourceData, meta interface{}) err
 	d.Set("display_text", n.Displaytext)
 	d.Set("cidr", n.Cidr)
 
+	// Read the tags and sort them on a map
```

Comment:
Small nit... 

I think this should be: `Read the tags and store them in a map` as the current comment gave me the impression you are trying to sort the tags, but maps are unordered.